reference/features/security.md

```diff
@@ -0,0 +1,272 @@
+# Security
+
+MeiliSearch allows you to protect your instances by using API keys. API keys give you fine-grained control over which users can access which indexes, endpoints, and routes.
+
+## Protecting a MeiliSearch instance
```

Comment:
When you set or change the master key, all existing keys are regenerated (they still exist but receive a new key value). Would be nice to drop this info towards the end of this section.